src/verify-and-receive.ts

```diff
@@ -35,9 +29,6 @@ export async function verifyAndReceive(
   return state.eventHandler.receive({
     id: event.id,
     name: event.name,
-    payload:
-      typeof event.payload === "string"
-        ? JSON.parse(event.payload)
-        : event.payload,
+    payload: JSON.parse(event.payload),
```

Comment:
Great that we are addressing: https://github.com/octokit/webhooks.js/issues/589#issuecomment-1376814798
I guess you could argue that it should be the `receive` function's responsibility to parse the `JSON`.